### DISCUSSION TITLE
Pandas Library (chpurse H2 pada jam 15.20)

### DISCUSSION THREAD

**Discussion by Daizane** [teamalgoritma/community#643] [Q&A] [answered]
pd.read_csv("data_input/rice.csv") untuk pengkodean tersebut menunjukan hasil ERROR tidak bisa terapliaksikan, gimana ya solusinya, pada file downlaod saya keterangan belakang nya bukan csv tapi XLS worksheet sehingga hanya bisa di buka di excel saja. sudah redownload file nya berkali2x tapi masih jg yang terbaca XLS Worksheet bukan CSV, udh buka dengan excel dan resave ke csv masih ndak bisa, 
![image](https://user-images.githubusercontent.com/128264111/232964584-c6039deb-5d03-4ca2-af68-4f6b65c30d6b.png)

**Comment by Daizane** [comment]
![image](https://user-images.githubusercontent.com/128264111/232964713-4c537f1a-d0c1-41af-aff0-eb681cb0c718.png)
![image](https://user-images.githubusercontent.com/128264111/232964742-47ec508b-e2fd-46c3-9e81-9ca2e1f7a1b8.png)
![image](https://user-images.githubusercontent.com/128264111/232964760-c24a1548-887e-420f-87b1-ce37824a9f30.png)

**Comment by finesaaa** [comment, marked as answer]
Dear @Daizane, bolehkah di screenshoot untuk full error bagian bawah?
Karena seharusnya jenis filenya benar
